src/Microsoft.ApplicationInsights.AspNetCore/Logging/Implementation/ApplicationInsightsLoggerFactoryExtensions.cs

```diff
@@ -50,11 +50,27 @@ public static ILoggerFactory AddApplicationInsights(this ILoggerFactory factory,
             IServiceProvider serviceProvider,
             Func<string, LogLevel, bool> filter)
         {
+            return factory.AddApplicationInsights(serviceProvider, filter, () => { });
```

Comment:
Can't you use 'null' instead of creating an empty delegate? Then it can be a default parameter. Handle null in ```AddLoggerCallback``` with ```callback?.Invoke()```